applications/PNS.md

```diff
@@ -0,0 +1,426 @@
+
+We have won Community Choice award in the 2nd Polkadot Hackathon held by Parity and OneBlock community in July, 2021. 
+
+We have presented PNS on Polkadot Decoded 2022. 
+
+Currently, PNS team has participated in Web3Bootcamp. 
+
+### **Team Code Repos**
+
+- [https://github.com/pnsproject](https://github.com/pnsproject)
+- [https://github.com/pnsproject/pns-sdk](https://github.com/pnsproject/pns-sdk)
+- [https://github.com/pnsproject/pns-contracts](https://github.com/pnsproject/pns-contracts)
+
+Team members:
+
+- [https://github.com/jiangplus](https://github.com/jiangplus)
+- [https://github.com/qieman](https://github.com/qieman)
+- [https://github.com/sg552](https://github.com/sg552)
+
+## **Development Status** and Security Audit **📖**
+
+The PNS contracts are built from the ground up with a more flexible and extensible design in mind. 
+
+We have finished the solidity contract development and got contracts audited by PeckShield. The audit report is available at the Github repo ([https://github.com/pnsproject/pns-contracts/tree/master/audit](https://github.com/pnsproject/pns-contracts/tree/master/audit)). We have fixed all the issues from the audit report.
+
+## **Development Roadmap 🔩**
+
+### Overview
+
+- Support multi tokens for payment of PNS registration and renewal.
+- Account ownership and social network identity integration with verifiable credentials
+- PNS HTTPS API
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):** 3
+- **Total Costs:** $26,000
+
+### **Milestone 1  —** Multi-token support for PNS Registration and Renewal
+
+- **Estimated duration:** 1 month
+- **FTE:** 2
+- **Costs:** 8,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT  |
+| 0b. | Documentation | We will provide documentation that explains how a user can register a domain with dot, ksm, aUSD, USDC.   |
+| 0c. | Tests | Core payment functions will be fully covered by unit tests to ensure functionality and robustness.  |
+| 1. | Multi-token support | Since Polkadot is a multi-chain ecosystem, users may have different assets on different chains. As a DID infrastructure, PNS serves users across all parachains. If PNS only supports one token, e.g. GLMR on Moonbeam, those without such tokens may find it difficult to use their DIDs. Thus, it is necessary to support multiple tokens to register PNS. This is not intended to promote a specific token, but to facilitate all Polkadot users at the ecosystem level. We will implement the functionality to accept dot, ksm to register PNS domains.  |
+| 2. | Interaction with Frontend and Payment ABI of price oracle | PNS registration and renewal fee is priced in USD. In order to display the price and receive the fee correctly, the PNS App should read the price of tokens from other oracles and allow users to pay various tokens to obtain their PNS.  |
```

Comment:
Did you already make plans on which oracle you're going to use and what the integration will look like?

Reply:
Hi @takahser,  we are using Chainlink data feeds as our price oracle to get the price of KSM/USD, DOT/USD and GLMR/USD.